python_modules/dagster-graphql/dagster_graphql_tests/graphql/test_asset_backfill.py

```diff
@@ -150,6 +151,36 @@ def test_launch_asset_backfill_read_only_context():
                 == "UnauthorizedError"
             )
 
+        location_name = main_repo_location_name()
+
+        # context with per-location permissions on the specific location succeeds
+        with define_out_of_process_context(
+            __file__,
+            "get_repo",
+            instance,
+            read_only=True,
+            read_only_locations={location_name: False},  # not read-only in this specific location
+        ) as read_only_context:
+            assert read_only_context.read_only
+            # launchPartitionBackfill
+            launch_backfill_result = execute_dagster_graphql(
+                read_only_context,
+                LAUNCH_PARTITION_BACKFILL_MUTATION,
+                variables={
+                    "backfillParams": {
+                        "partitionNames": ["a", "b"],
+                        "assetSelection": [key.to_graphql_input() for key in all_asset_keys],
```

Comment:
just to make sure, `all_asset_keys` here is the set of all asset keys in `location_name`?

Reply:
are we de-selecting any assets with the subselection (e.g. assets in a read-only location?)